python/pyspark/sql/session.py

```diff
@@ -432,14 +430,9 @@ def createDataFrame(self, data, schema=None, samplingRatio=None):
             ``byte`` instead of ``tinyint`` for :class:`pyspark.sql.types.ByteType`. We can also use
             ``int`` as a short name for ``IntegerType``.
         :param samplingRatio: the sample ratio of rows used for inferring
+        :param verifySchema: verify data types of every row against schema.
         :return: :class:`DataFrame`
 
-        .. versionchanged:: 2.0
```

Comment:
@davies, I'm also slightly confused by this documentation change since it looks like the new 2.x behavior of wrapping single-field datatypes into structtypes and values into tuples is preserved by this patch. Could you clarify?